### ISSUE TITLE
Transferring subscriptions, etc. to a new Mac/new User?

### ISSUE THREAD

**Issue by timwux** [mackworth/cTiVo#505]
<Obligatory apologies in advance if I'm a dolt and this function is right in front of me.>

Is transferring subscriptions—and maybe even the transfer queue—between Users on a Mac (or to a new Mac) possible? The only Library item that I could find that was recently updated was the HTTPStorages folder, so no go there.

Thanks for continuing to release such high quality work.

tim

**Comment by mackworth** [comment]
Thanks for the nice compliment!

All of that (queue, subscriptions etc) is stored in the preferences file. (So the former user’s `~/Library/Preferences/com.ctivo.ctivo.plist` )

**Comment by timwux** [comment]
Of course! Your software is wonderful and makes a slowly decaying product still relevant to us.

I copied the <500KB file from the original user to the new and relaunched the CTiVo 3.5.2 (1139) and was presented with a mirror of everything I needed.

Thanks for the quick (and easy) fix!

Stay safe,
Tim
